contracts/samples/gnosis/EIP4337Manager.sol

```diff
@@ -163,8 +163,8 @@ contract EIP4337Manager is IAccount, GnosisSafeStorage, Executor {
         sig[2] = bytes1(uint8(1));
         sig[35] = bytes1(uint8(1));
         uint256 nonce = uint256(IEntryPoint(manager.entryPoint()).getNonce(address(safe), 0));
-        UserOperation memory userOp = UserOperation(address(safe), nonce, "", "", 0, 1000000, 0, 0, 0, "", sig);
-        UserOperation[] memory userOps = new UserOperation[](1);
+        PackedUserOperation memory userOp = PackedUserOperation(address(safe), nonce, "", "", bytes32(bytes16(uint128(0x0f4240))), 0, 0, 0, "", sig);
```

Comment:
afaik this packing is not needed: the userop is without paymaster, so `bytes32("")` is enough (afaik, its the shortest way in solidity to write it, better than bytes32(uint(0)), since byte32(0) is not allowed..